django/db/__init__.py

```diff
@@ -59,3 +62,13 @@ def close_old_connections(**kwargs):
 
 signals.request_started.connect(close_old_connections)
 signals.request_finished.connect(close_old_connections)
+
+
+# Register an event to close all connections before forking a process.
+# Only register if the platform supports forking.
+def close_all_connections(**kwargs):
+    connections.close_all()
+
+
+if platform.system() == "Linux":
```

Comment:
Maybe just do `if hasattr(os, 'register_at_fork'):`? macOS is an UNIX OS too, and `register_at_fork` is "Availability: UNIX".